src/components/Stake.svelte

```diff
@@ -94,38 +91,40 @@
   }
 </script>
 
-<div class:hidden={false} class="action-card basis-1/2">
-  <p class="action-card-title">
-    Stake to Provider Id {providerId}
-  </p>
-  <div>
-    <ol class="ml-4 mt-4 list-decimal font-light">
-      <li>Ensure the control key has a FRQCY balance.</li>
-      <li>Click 'Stake'</li>
-      <li>This will require 1 signature to send the transaction.</li>
-    </ol>
-  </div>
-  <form class="mt-8">
-    <KeySelection
-      component="SelectControlKey"
-      selectLabel="Key to Stake From"
-      bind:selectedOption={selectedKey}
-      {validAccounts}
-      classOverrides="border-2 rounded-lg"
-    />
-    <div class="mt-6">
-      <label for="stakingInput">Amount to Stake in <span class="units">{token}</span></label>
-      <div class="input-container mt-2">
-        <input type="number" id="stakingInput" value="1" on:input={handleInput} />
+<Modal id="stake-to-provider" {isOpen} {close}>
+  <BlockSection title="Stake to Provider">
```

Comment:
BlockSection has padding and some styling that is not what we see in the Figma for the Modal. If you want to reusable styling here, I would recommend taking the styling from <ConnectProvider />, putting it in <Modal />, and then using slots to bring in the title and body.